src/languages/es.js

```diff
@@ -7,6 +7,7 @@ export default {
         yes: 'Sí',
         no: 'No',
         ok: 'OK',
+        buttonConfirm: 'Ok, entendido',
```

Comment:
Not sure how the process goes with translations. These texts should probably be reviewed with caution!